pkg/sources/postman/postman.go

```diff
@@ -60,8 +60,18 @@ func (s *Source) addKeywords(keywords []string) {
 }
 
 func (s *Source) addKeyword(keyword string) {
+	// fast check
 	if _, ok := s.DetectorKeywords[keyword]; ok {
 		s.keywords[keyword] = struct{}{}
+		return
+	}
+
+	// slow check. This is to handle the case where the keyword is a substring of a detector keyword
+	// e.g. "datadog-token" is a variable key in postman, but "datadog" is a detector keyword
+	for k := range s.DetectorKeywords {
+		if strings.Contains(keyword, k) {
+			s.keywords[k] = struct{}{}
```

Comment:
In this example, do we still want "datadog-token" or just "datadog". I only ask b/c character distance b/t keyword and secret could be impacted.

Reply:
We want to store just `datadog` as `datadog-token` doesn't provide additional useful information.

Reply:
Ah yes. And looks like that's what you originally did. Read that wrong.